docs/app/i18n/en-US/pages.js

```diff
@@ -86,25 +86,25 @@ export default {
     },
     flexible: {
       title: 'Flexible height and text size',
-      desc: 'Maybe the focus of your page is the title, following by the content itself. The page that shows details of a single contact is a great example of this scenario. So you can use flexible behaviour to give higher piority to your content after scrolling.',
+      desc: 'Maybe the focus of your page is the title, followed by the content itself. A page that shows the details of a single contact is a great example of this scenario. So you can use flexible behaviour to give higher piority to your content after scrolling.',
       flexible: 'Flexible'
     },
     lastRow: {
       title: 'The last toolbar row fixed',
-      desc: 'When your page has some primary navigation buttons, such as tabs and others, you may want them to be always available. You can make the second row of your toolbar always visible, hiding the first one when it\'s not needed. As an example, the WhatsApp main page has this behaviour.',
+      desc: 'When your page has some primary navigation buttons, such as tabs and others, you may want them to always be available. You can make the second row of your toolbar always visible, hiding the first one when it\'s not needed. As an example, the WhatsApp main page has this behaviour.',
       fixed: 'Navigation fixed'
     },
     overlap: {
       title: 'Flexible space with overlapping content',
-      desc: 'If you want to give focus on the content cards you may want to use the overlap. It\'s just an awesome effect!',
+      desc: 'If you want to give focus to the content cards, you may want to use the overlap. It\'s just an awesome effect!',
       content: 'Overlapping content'
     },
     apiApp: {
-      desc: 'In both examples we have 3 distinct areas: Toolbar, Drawer and Content. You should create them using the following tags:',
+      desc: 'In these examples we have 3 distinct areas: Toolbar, Drawer and Content. You should create them using the following tags:',
       toolbar: 'Creates the toolbar accepting any options from the md-toolbar',
-      drawer: 'Places the drawer panel, that also accepts any options from the md-drawer. You can have a left or right placed drawers. Or both.',
+      drawer: 'Places the drawer panel, that also accepts any options from the md-drawer. You can have left or right placed drawers. Or both.',
```

Comment:
Drawer could only be placed one at most at once for now. Would you fix it?

Reply:
@VdustR Sure thing. I'll push a commit in a moment...